projects/plugins/search/tests/e2e/helpers/search-helper.js

```diff
@@ -88,8 +88,8 @@ export async function searchAPIRoute( page ) {
 		}
 
 		// deal with filtering: only works with one category and one tag by filtering the results array
-		const category = params.get( 'filter[bool][must][0][term][category.slug]' );
-		const tag = params.get( 'filter[bool][must][0][term][tag.slug]' );
+		const category = params.get( 'filter[bool][must][0][bool][should][0][term][category.slug]' );
+		const tag = params.get( 'filter[bool][must][0][bool][should][0][term][tag.slug]' );
```

Comment:
I think you may need to set a minimum_should_match param also otherwise these filters won't actually filter anything out. The default is zero: https://www.elastic.co/guide/en/elasticsearch/reference/8.3/query-dsl-bool-query.html#bool-min-should-match

Reply:
Added in 168fa1f364